lib/metrics.js

```diff
@@ -7,6 +7,7 @@ DESCRIPTIONS.set('read', 'Read hypercore blocks from hard disk')
 DESCRIPTIONS.set('write', 'Write hypercore blocks to hard disk')
 DESCRIPTIONS.set('download_read_stream', 'Download hypercore blocks using a readStream')
 DESCRIPTIONS.set('download', 'Download hypercore blocks using a download range')
+DESCRIPTIONS.set('demo', 'demo')
```

Comment:
Rename to `read_hyperdrive' and give a description

Reply:
added a description